caiman_napari/cnmf_results.py

```diff
@@ -0,0 +1,63 @@
+from caiman.utils.visualization import get_contours as caiman_get_contours
+from .utils import *
+
+
+def show_results(cnmf_obj, viewer):
```

Comment:
I would put this in the same module that is referred by the `ALGO_MODULES` (`_cnmf.py`) and call it `load_output()`. And the same for mcorr. So it's uniform.

The function should take the viewer as the first argument since that will be uniform across mcorr, cnmf, cnmfe etc. and then the `DataFrame` row (pandas `Series` object) which contains all the relevant information.